plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/app/PluginStrings.js

```diff
@@ -38,12 +38,13 @@ Ext.define('NX.coreui.app.PluginStrings', {
     BROWSE_SEARCH_SAVE_NAME: 'Filter name',
     BROWSE_SEARCH_SAVE_DESCRIPTION: 'Filter description',
     BROWSE_SEARCH_COMPONENTS_CRITERIA_KEYWORD: 'Keyword',
-    BROWSE_SEARCH_COMPONENTS_CRITERIA_ARTIFACT_ID: 'Artifact ID',
+    BROWSE_SEARCH_COMPONENTS_CRITERIA_NAME: 'Name',
     BROWSE_SEARCH_COMPONENTS_CRITERIA_CLASSIFIER: 'Classifier',
     BROWSE_SEARCH_COMPONENTS_CRITERIA_CLASS_NAME: 'Class name',
     BROWSE_SEARCH_COMPONENTS_CRITERIA_FORMAT: 'Format',
-    BROWSE_SEARCH_COMPONENTS_CRITERIA_GROUP_ID: 'Group ID',
+    BROWSE_SEARCH_COMPONENTS_CRITERIA_GROUP: 'Group',
```

Comment:
For CLM we adopted 'namespace' to represent the top level of any given component format hierarchy. Might be worthwhile to start thinking cross-format nomenclature here, same as for artifactId above.